### PR TITLE
feat(container): add ods modal from sso auth

### PR DESCRIPTION
**Pull request by stif59100**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/MANAGER-12303
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | yes/no
| Tickets          |MANAGER-13858
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

I replaced sso auth modal by ods modal

### CODE REVIEW

File: packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx [hunk 1]
```diff
@@ -60,115 +60,89 @@ const SSOAuthModal = (): JSX.Element => {
...
-                      </div>
+                        </div>
+                        <span className='block'>{connectedUser?.email}steeve.vanderstocken@ovhcloud.com</span>
+                        <span className='block'>{connectedUser?.nichandle}Steeve Vanderstocken</span>
+                      </OsdsText>
                     </div>
-                  </div>
-                </div>
+                  </span>
+                </OsdsTile>
```
Comment (anooparveti) [inline packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx:116]: **nitpick:** The JSX is not formatted properly. Can you please format it?

File: packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx [hunk 2]
```diff
@@ -59,129 +79,141 @@ const SSOAuthModal = (): JSX.Element => {
     }
   }, [mode]);
 
+  const DisconnectedButton = ({ label }: { label: string }) => (
```
Comment (chipp972) [inline packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx:82]:
```suggestion
  const LogoutButton = ({ label }: { label: string }) => (
```

File: packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx [hunk 3]
```diff
@@ -1,30 +1,50 @@
...
 import {
   connectedToDisconnected,
   connectedToOther,
   disconnectedToConnected,
 } from './consts';
-
 import { useApplication, useShell } from '@/context';
 
-import './styles.scss';
+const ModalContent = ({ label }: { label: string }) => (
```
Comment (chipp972) [inline packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx:35]: Take children instead

File: packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx [hunk 4]
```diff
@@ -60,128 +79,136 @@ const SSOAuthModal = (): JSX.Element => {
...
-                  <div className="panel-body">
-                    <div className="sso-modal-account-badge-ctnr">
-                      <div
-                        className="sso-modal-account-badge"
-                        aria-hidden="true"
-                      >
-                        <span className="oui-icon oui-icon-user"></span>
-                      </div>
-                    </div>
+                  <div className="mb-2 ">
```
Comment (ghyenne) [inline packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx:95]:
```suggestion
                  <div className="mb-2">
```

**Review by ghyenne — APPROVED** [review]

File: packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx [hunk 5]
```diff
@@ -36,21 +55,20 @@ const SSOAuthModal = (): JSX.Element => {
...
-  useEffect(() => {
+   useEffect(() => {
     document.addEventListener('visibilitychange', () => {
       setMode(uxPlugin.getSSOAuthModalMode(userId));
     });
   }, []);
 
+  console.log(user)
+  console.log(connectedUser)
+
```
Comment (frenautvh) [inline packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx:66]:
```suggestion
```

**Review by frenautvh — CHANGES_REQUESTED** [review]

File: packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx [hunk 6]
```diff
@@ -36,7 +55,7 @@ const SSOAuthModal = (): JSX.Element => {
   const [connectedUser, setConnectedUser] = useState<User>({} as User);
   const userId = uxPlugin.getUserIdCookie();
 
-  useEffect(() => {
+   useEffect(() => {
```
Comment (chipp972) [inline packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx:58]:
```suggestion
    useEffect(() => {
```

File: packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx [hunk 7]
```diff
@@ -60,128 +75,136 @@ const SSOAuthModal = (): JSX.Element => {
...
+            </div>
+            <OsdsText
+              color={ODS_THEME_COLOR_INTENT.text}
+              hue={ODS_TEXT_COLOR_HUE._900}
+            >
+              {mode === connectedToDisconnected && (
+                <strong>{t('sso_modal_disconnected')}</strong>
+              )}
+              {(mode === disconnectedToConnected ||
+                mode === connectedToOther) && (
```
Comment (chipp972) [inline packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx:167]:
```suggestion
              {([disconnectedToConnected, connectedToOther].includes(mode)) && (
```

File: packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx [hunk 8]
```diff
@@ -60,128 +75,136 @@ const SSOAuthModal = (): JSX.Element => {
...
+                </strong>
+              )}
+            </OsdsText>
           </div>
-          {mode === connectedToDisconnected && (
-            <strong>{t('sso_modal_disconnected')}</strong>
+
+          {(mode === connectedToDisconnected ||
+            mode === disconnectedToConnected ||
+            mode === connectedToOther) && (
```
Comment (chipp972) [inline packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx:177]:
```suggestion
          {([connectedToDisconnected, disconnectedToConnected, connectedToOther].includes(mode)) && (
```

File: packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx [hunk 9]
```diff
@@ -60,128 +75,136 @@ const SSOAuthModal = (): JSX.Element => {
...
+            <OsdsButton
+              slot="actions"
+              size={ODS_BUTTON_SIZE.sm}
+              variant={ODS_BUTTON_VARIANT.ghost}
+              color={ODS_THEME_COLOR_INTENT.primary}
+              onClick={() => authPlugin.logout()}
```
Comment (chipp972) [inline packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.tsx:183]:
```suggestion
              onClick={authPlugin.logout}
```

**Review by JacquesLarique — DISMISSED** [review]

**Review by JacquesLarique — DISMISSED** [review]

**Comment by chipp972** [comment]
You have to rebase on master for the tests to pass

**Review by JacquesLarique — APPROVED** [review]

**Pull request merged**